.github/workflows/deploy-production.yaml

```diff
@@ -0,0 +1,34 @@
+name: Deploy Production
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy-backend:
+    name: Deploy Backend to Fly
+    environment: Production
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+  deploy-frontend:
```

Comment:
I assume there was a reason for this? It's just to ensure the backend finishes first?

Reply:
yup as discussed offline